api/lib/application/authentication/oidc/oidc-controller.js

```diff
@@ -129,8 +129,8 @@ const authenticateUser = async function (
   } else {
     const message = "L'utilisateur n'a pas de compte Pix";
     const responseCode = 'SHOULD_VALIDATE_CGU';
-    const { authenticationKey, givenName, familyName } = result;
-    const meta = { authenticationKey, givenName, familyName };
+    const { authenticationKey, givenName, familyName, email } = result;
+    const meta = { authenticationKey, givenName, familyName, email };
```

Comment:
C'est peut être un peu trop, mais je pense que l'on ne devrait pas retourner une valeur **undefined**. Du moins, sur Pix App, `email` sera toujours undefined.

```suggestion
    const { authenticationKey, givenName, familyName, email } = result;
    const meta = { authenticationKey, givenName, familyName, email };
    if (email) Object.assign(meta, { email });
```

On peut aussi envisager de séparer la partie authentification OIDC `/api/admin` et `/api`

Reply:
Au vu de l'erreur, est-ce que finalement cette erreur n'aurait pas plutôt sa place côté usecase et passer par le système déjà existant  (error-manager, etc.) ?

Reply:
@Steph0 j'ai l'impression que tu n'as pas mis le commentaire au bon endroit.

Reply:
done !